tools/cypress/update_integrity.sh

```
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Generate integrity hashes for the cypress_version in MODULE.bazel by running:
+#   bazel run @aspect_rules_cypress//scripts:mirror_releases -- <VERSION>
+# Then write the generated platform hashes into the cypress_integrity block.
+
+set -euo pipefail
+
+readonly SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)